webapp/src/vad.ts

```diff
@@ -18,8 +18,8 @@ export default class VoiceActivityDetector extends EventEmitter {
             freqRange: [80, 400],
             noiseCaptureMs: 1000,
             noiseMultiplier: 1.5,
-            activityThreshold: 4,
-            activityCounterMax: 10,
+            activityThreshold: 5,
+            activityCounterMax: 30,
```

Comment:
In practical terms this change will mean roughly 1 second more leniency before considering voice over.